File: examples-sdk/swift/028-basic-sse-hub-streaming/main.swift
// 028-basic-sse-hub-streaming — Swift SDK equivalent
// Compile: vil compile --from swift --input 028-basic-sse-hub-streaming/main.swift --release

let server = VilServer(name: "sse-hub-demo", port: 8080)
let events = ServiceProcess(name: "events")
events.endpoint(method: "POST", path: "/publish", handler: "publish")
events.endpoint(method: "GET", path: "/stream", handler: "stream")
events.endpoint(method: "GET", path: "/stats", handler: "stats")
server.service(events)
server.compile()

File: examples-sdk/kotlin/028-basic-sse-hub-streaming/main.kt
// 028-basic-sse-hub-streaming — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 028-basic-sse-hub-streaming/main.kt --release

fun main() {
    val server = VilServer("sse-hub-demo", 8080)
    val events = ServiceProcess("events")
    events.endpoint("POST", "/publish", "publish")
    events.endpoint("GET", "/stream", "stream")
    events.endpoint("GET", "/stats", "stats")
    server.service(events)
    server.compile()
}
